docs/CHILI-GraFx/trust/AI/index.md

```
@@ -0,0 +1,50 @@
+## Data usage and training
+
+- Customer data is only processed to deliver the Services
+- Customer data is **not** used to train, enrich, or improve AI models for any other purpose
+- There is no cross-customer data usage
+- There is no reuse of customer data outside the scope of the Services
+
+## Security, Compliance, and AI Governance
+
+AI components are subject to the same rigorous security and compliance standards as the rest of the CHILI GraFx platform. All AI-powered functionality is managed within our ISO-certified framework, ensuring that safeguards and controls are strictly aligned with the risk level of each specific capability.
+
+**AI Tooling and Dynamic Evaluation**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)